features/serviceworker-blocking.json

```diff
@@ -0,0 +1,11 @@
+{
```

Comment:
I mean up to you, but perhaps `serviceworker-initiated-requests.json` would be a better name? Most of the other feature names seem to be the thing they're blocking or protecting against. Also, I figure "ServiceWorker blocking" might be a kind of confusing name, making folks think we are looking to block ServiceWorkers entirely?